src/platform-aws.c

```diff
@@ -681,6 +681,31 @@ int platform_config_endpoint(struct fi_info *info, struct fid_ep* endpoint) {
 	 * ordering.  If we're not expecting ordering, we don't really
 	 * care if ordering is on or off for the endpoint.
 	 */
+
+	/* TODO: This is a temporary hack to disable setting
```

Comment:
The above comment on the code should be above the `if (need_ordering || !nccl_proto_configured)` check.